_pytest/mark/structures.py

```diff
@@ -365,3 +372,30 @@ def __len__(self):
 
     def __repr__(self):
         return "<NodeKeywords for node %s>" % (self.node, )
+
+
+@attr.s(cmp=False, hash=False)
+class NodeMarkers(object):
+    """
+    internal strucutre for storing marks belongong to a node
```

Comment:
Should this be added to the docs in `reference.rst`?

Reply:
i think so, it should be noted that it currently cant be accessed via public attributes

Reply:
If that's the case, then it doesn't need adding to the docs does it?